agents/logsuploader/uploader.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/microsoft/ApplicationInsights-Go/appinsights"
+)
+
+func tryUpload(maybeJSON string, client appinsights.TelemetryClient) bool {
+	fmt.Println("attempting to parse and load", maybeJSON)
+	result := false
+
+	var deserialized Event
+
+	err := json.Unmarshal([]byte(maybeJSON), &deserialized)
+
+	if err == nil {
+		event := createEvent(deserialized)
+		client.Track(event)
```

Comment:
Is there client.Flush()? Let's make an effort to send it right away if available.

Reply:
done